src/cdk/a11y/aria-describer/aria-reference.ts

```diff
@@ -6,7 +6,7 @@
  * found in the LICENSE file at https://angular.io/license
  */
 
-/** IDs are deliminated by an empty space, as per the spec. */
+/** IDs are delimited by an empty space, as per the spec. */
 const ID_DELIMINATOR = ' ';
```

Comment:
Also, maybe this private constant, `ID_DELIMINATOR`, should be something else. Maybe `ID_SEPARATOR`?

Reply:
It should be `ID_DELIMITER` if you want to fix that in this PR

Reply:
Done.